internal/bminventory/inventory.go

```diff
@@ -839,7 +837,6 @@ func (b *bareMetalInventory) GenerateClusterISO(ctx context.Context, params inst
 	updates["image_ssh_public_key"] = params.ImageCreateParams.SSHPublicKey
 	updates["image_created_at"] = strfmt.DateTime(now)
 	updates["image_expires_at"] = strfmt.DateTime(now.Add(b.Config.ImageExpirationTime))
-	updates["image_generator_version"] = b.Config.ImageBuilder
```

Comment:
For a followup, but I assume we can eventually remove this column, right?

Reply:
from the swagger as well.